fri/src/verifier.rs

```diff
@@ -47,6 +48,13 @@ pub(crate) fn verify<FC: FriConfig>(
         return Err(VerificationError::InvalidProofShape);
     }
 
+    // Check PoW.
+    challenger.observe(proof.pow_witness);
+    let pow_bits = challenger.sample_bits(config.proof_of_work_bits());
+    if pow_bits != 0 {
```

Comment:
Can this be shortened with `if !challenger.check_witness(...) {`?

Reply:
Ah good idea!